contrib/pg_tde/documentation/docs/variables.md

```diff
@@ -4,44 +4,55 @@ The `pg_tde` extension provides GUC variables to configure the behaviour of the
 
 ## pg_tde.wal_encrypt
 
+**Type** - boolean <br>
+**Default** - off
+
 A `boolean` variable controlling if WAL writes are encrypted or not.
 
 Changing this variable requires a server restart, and can only be set at the server level.
 
-WAL encryption is controlled globally, if enabled all WAL writes are encrypted.
-
-This variable only controls new writes to the WAL, it doens't affect existing WAL records.
-`pg_tde` is always capable of reading existing encrypted WAL records, as long as the keys used for the encryption are still available.
+WAL encryption is controlled globally. If enabled, all WAL writes are encrypted in the entire PostgreSQL cluster.
 
+This variable only controls new writes to the WAL, it doesn't affect existing WAL records.
 
+`pg_tde` is always capable of reading existing encrypted WAL records, as long as the keys used for the encryption are still available.
 
-Enabling WAL encryption requires a configured global principal key.
-
-The default value is false.
+Enabling WAL encryption requires a configured global principal key. Refer to the [WAL encryption configuration](wal-encription.md) documentation for more information.
 
 ## pg_tde.enforce_encryption
 
-A `boolean` variable controlling if creation of new not encrypted tables is enabled or not.
+**Type** - boolean <br>
+**Default** - off
 
-This can be set at global, database, user or session level, but changing the value requires superuser permissions.
+A `boolean` variable controlling if the creation of new not encrypted tables is enabled or not.
 
 If enabled, `CREATE TABLE` statements will fail unless they use the `tde_heap` access method.
 
 Similarly, `ALTER TABLE <x> SET ACCESS METHOD` is only allowed, if the access method is `tde_heap`.
 
-Other DDL operations are still allowed, for example other `ALTER` commands are allowed on unencrypted tables, as long as the access method isn't changed.
+Other DDL operations are still allowed. For example other `ALTER` commands are allowed on unencrypted tables, as long as the access method isn't changed.
+
+You can set this variable at the following levels. 
 
-The default value is false.
+* global - for the entire PostgreSQL cluster
+* database - for the current database
```

Comment:
for specific databases

Reply:
How to know what databases is it set for?

Reply:
We don't have to know, it's managed by postgres: https://www.postgresql.org/docs/current/sql-alterdatabase.html

It's the `ALTER DATABASE ... SET variable = value;` syntax.

Reply:
This is also why I didn't write a too detailed description about it. This is a common postgresql concept for most GUC variables, we shouldn't have to explain it to anybody familiar with postgres.